query.go

```diff
@@ -203,3 +246,29 @@ func applyHeaders(query *Query, headers http.Header) *Query {
 
 	return query
 }
+
+// sanitizeLabelName removes all invalid chars from the label name.
+// If the label name is empty or contains only invalid chars, it
+// will return an error.
+func sanitizeLabelName(name string) (string, bool) {
```

Comment:
Should we add test for this function?

Reply:
this was copied from https://github.com/grafana/grafana/blob/main/pkg/infra/metrics/metricutil/utils.go#L14

Reply:
Should we then copy test also?
https://github.com/grafana/grafana/blob/main/pkg/infra/metrics/metricutil/utils_test.go#L11-L33